### ISSUE TITLE
Possible InputMismatchException

### ISSUE THREAD

**Issue by arkulowski** [NCRaider1/SchoolProblems#2]
When you use the input.nextInt() to get the graduation yearit is expecting an integer. If the user does not give an integer it would result in a InputMismatchException. You could make sure that the user gives an integer before calling the input.nextInt() by using input.hasNextInt().

**Comment by AlecGoriup** [comment]
I agree, this would be a great issue to resolve if we were to add more user input error exceptions
